crates/sui-framework/packages/sui-system/sources/sui_system.move

```diff
@@ -1,6 +1,37 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+/// Sui System State Type Upgrade Guide
```

Comment:
Do we want to mention somewhere that for small upgrades we can just add fields to the bag, and don't need to introduce new types?